File: LeetCode/python3/867.py
class Solution:
    def transpose(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        row, col = len(A), len(A[0])
        res = [[0 for i in range(row)] for j in range(col)]
        for i in range(col):
           for j in range(row):
               res[i][j] = A[j][i]
        return res

File: LeetCode/cpp/867.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>> &A) {
        auto row = A.size();
        if (row == 0)
            return {};
        vector<vector<int>> ret;
        auto col = A[0].size();
        for (int i = 0; i < col; ++i) {
            vector<int> temp;
            for (int j = 0; j < row; ++j) {
                temp.push_back(A[j][i]);
            }
            ret.push_back(temp);
        }
        return ret;
    }
};
